internal/observability/event/eventer_config.go

```diff
@@ -6,24 +6,15 @@ import (
 
 // EventerConfig supplies all the configuration needed to create/config an Eventer.
 type EventerConfig struct {
-	AuditDelivery       DeliveryGuarantee // AuditDelivery specifies the delivery guarantees for audit events (enforced or best effort).
-	ObservationDelivery DeliveryGuarantee // ObservationDelivery specifies the delivery guarantees for observation events (enforced or best effort).
-	AuditEnabled        bool              // AuditEnabled specifies if audit events should be emitted.
-	ObservationsEnabled bool              // ObservationsEnabled specifies if observation events should be emitted.
-	Sinks               []SinkConfig      // Sinks are all the configured sinks
+	AuditEnabled        bool         `hcl:"audit_enabled"`        // AuditEnabled specifies if audit events should be emitted.
+	ObservationsEnabled bool         `hcl:"observations_enabled"` // ObservationsEnabled specifies if observation events should be emitted.
+	Sinks               []SinkConfig `hcl:"sinks"`                // Sinks are all the configured sinks
 }
 
-// validate will validate the config. BTW, a config isn't required to have any
+// Validate will Validate the config. BTW, a config isn't required to have any
```

Comment:
```suggestion
// Validate will Validate the config. A config isn't required to have any
```
I love the BTW but wondering if it should just be?